### PR TITLE
I add css animate

### PR DESCRIPTION
**Pull request by zedutra**

### CODE REVIEW

File: desktop-tools.md [hunk 1]
```diff
@@ -88,3 +88,4 @@
 [Wireshark](https://www.wireshark.org/) | Wireshark is an open source, network protocol analyzer.
 [yEd](https://www.yworks.com/products/yed) | yEd is a powerful desktop application that can be used to quickly and effectively generate high-quality diagrams.
 [portainer](https://github.com/portainer/portainer) | Portainer is a lightweight management UI which allows you to easily manage your different Docker environments
+[CSS Animate](https://cssanimate.com/) | Online tool for creating native CSS3 Keyframes Animation
```
Comment (lucasgdb) [inline desktop-tools.md:91]: is it a desktop tool or a web tool?

**Review by lucasgdb — CHANGES_REQUESTED** [review]

File: desktop-tools.md (hunk 1 above)
Comment (zedutra) [reply desktop-tools.md:91]: Web tool

Comment (lucasgdb) [reply desktop-tools.md:91]: it is on "desktop-tools.md" file. Please, move it to "web-tools.md" file.

Comment (zedutra) [reply desktop-tools.md:91]: I'm sorry, i didn't pay attention to this.

**Review by lucasgdb — APPROVED** [review]

**Pull request merged**